config/policy-webhook.yaml

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+---
```

Comment:
this was needed in order to add the separator, otherwise the main manifest was invalid